code/game/objects/effects/spiders.dm

```diff
@@ -227,17 +228,29 @@
 			var/mob/living/simple_animal/hostile/poison/giant_spider/S = new grow_as(loc)
 			S.faction = faction.Copy()
 			S.master_commander = master_commander
-			if(player_spiders && !selecting_player)
+			if((prob(15)) && player_spiders && !selecting_player && !eventspiderling) // If the spider who laid the eggs was a player, slightly better chance.
 				selecting_player = 1
 				spawn()
-					var/list/candidates = pollCandidates("Do you want to play as a spider?", ROLE_GSPIDER, 1)
+					var/list/candidates = pollCandidates("Do you want to play as a giant spider?", ROLE_GSPIDER, 1, 150)
 
 					if(candidates.len)
 						var/mob/C = pick(candidates)
 						if(C)
 							S.key = C.key
-							if(S.master_commander)
-								to_chat(S, "<span class='biggerdanger'>You are a spider who is loyal to [S.master_commander], obey [S.master_commander]'s every order and assist [S.master_commander.p_them()] in completing [S.master_commander.p_their()] goals at any cost.</span>")
+							if(S.master_commander) // Born from a servant sentient spider, gotta follow the commander's orders.
+								to_chat(S, "<span class='biggerdanger'>You are a giant spider who is loyal to [S.master_commander], obey [S.master_commander]'s every order and assist [S.master_commander.p_them()] in completing [S.master_commander.p_their()] goals at any cost.</span>")
+							else // Born from a sentient spider who isn't loyal to anybody, do the thing giant spiders do.
+								to_chat(S, "<span class='biggerdanger'>You are a smart giant spider! Use your sentiency to your advantage, and help the spiders take over!</span>")

```

Comment:
maybe the word sentient instead of smart would be better in this case

Reply:
I thought of that... But if they were really sentient, they would know not to fight the crew anymore.
In this case... They are just slightly smarter than nornal spiders. Not enough to change the way they think.